### ISSUE TITLE
Pabot support

### ISSUE THREAD

**Issue by tomaspekarovic** [robocorp/robotframework-lsp#725]
**Is your feature request related to a problem? Please describe.**
I'm running a lot of tests locally from my computer. I have own library, so changes are directly reflected into lot of tests. So If I need to test for example one full suite, the plugin can only run it via `robot` command, so one at a time.

**Describe the solution you'd like**
Just add support to replace `robot` command with `pabot` to be able to run tests in parallel.

**Describe alternatives you've considered**
I'm just running tests from cmd line for now, but it will not update results in the  `Testing` tab in visual studio code

**Comment by fabioz** [comment]
I'm closing this one as it's a duplicate of: https://github.com/robocorp/robotframework-lsp/issues/562.

-- please upvote that issue if you're interested in this feature.
